eng/pipelines/common/setup-jdk.yml

```diff
@@ -9,6 +9,8 @@ steps:
       jdkPath="${{ parameters.jdkFolder }}"
       echo "Using provided JDK folder: $jdkPath"
     else
+      echo "Installed JDKs:"
+      /usr/libexec/java_home -V
```

Comment:
This has nothing to do with this change, but coincidentally build agents were failing on this point and I decided to add some debug info so we can see what is going on.

I still think this is good to have, so I'll just make it part of this PR